packages/wallet-sdk/src/store/store.ts

```diff
@@ -173,6 +194,18 @@ export const subAccounts = {
   },
 };
 
+export const spendLimits = {
+  get: () => sdkstore.getState().spendLimits,
+  set: (spendLimits: SpendLimit[]) => {
+    sdkstore.setState({ spendLimits });
+  },
+  clear: () => {
+    sdkstore.setState({
+      spendLimits: [],
```

Comment:
@fan-zhang-sv 

I wonder if we should match the config and store these by chainId to be more future facing

```ts
Record<chainId, SpendLimit[]>
```